### PR TITLE
Created example of how to make a simple inbound TTS

### PR DESCRIPTION
**Pull request by tommorris**
Using Sinatra, for the building block on receiving SMS

### CODE REVIEW

File: Gemfile [hunk 1]
```diff
@@ -2,7 +2,7 @@ source 'https://rubygems.org'
 
 gem 'nexmo', '4.5.0'
 gem 'dotenv', '2.1.1'
-gem 'faye/websocket', '0.10.7'
+gem 'faye-websocket', '0.10.7'
```
Comment (ChrisGuzman) [inline Gemfile]: Interesting The [faye's readme](https://github.com/faye/faye-websocket-ruby) says `gem install faye-websocket` but `require 'faye/websocket'` for use in your files. Did you check if the websocket examples run after making the change?

**Comment by tommorris** [comment]
@ChrisGuzman I *think* I've resolved the Gemfile merge conflict. Can you look at the codeclimate. Once this gets merged, I'll add it to docs.

File: voice/inbound_tts_call.rb [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+  [{
+    action: "talk",
+    text: "Hello world"
+  }].to_json
+end
+
+post '/callback' do
+  content_type :json
+  if params != {}
+    puts params
```
Comment (ChrisGuzman) [inline voice/inbound_tts_call.rb]: This isn't going to print out the json body. To do that you need to do `puts JSON.parse(request.body.read)`

File: voice/inbound_tts_call.rb [hunk 3]
```diff
@@ -0,0 +1,18 @@
...
+  end
+  {}.to_json
```
Comment (ChrisGuzman) [inline voice/inbound_tts_call.rb]: What is this doing? Just returning a 200?

Comment (tommorris) [inline voice/inbound_tts_call.rb]: @ChrisGuzman Yep, and an empty JSON document.

File: voice/inbound_tts_call.rb (hunk 2 above)
Comment (tommorris) [inline voice/inbound_tts_call.rb]: @ChrisGuzman Fixed.

**Pull request merged**